src/app/components/action-bar/action-bar.jsx

```diff
@@ -7,6 +7,7 @@ import {
   ButtonGroup,
   ThemeProvider,
   makeStyles,
+  Typography,
```

Comment:
I don't think this ChplActionBar is used anywhere in the feature we're working on, so I'm not sure why it's getting modified here. Any changes to this would need to be tested, but won't show up in any of the features that are related to this code